pkg/operator/collection.go

```diff
@@ -494,8 +494,6 @@ func (r *collectionReconciler) makeCollectorConfig(ctx context.Context, spec *mo
 			logger.Info("ClusterNodeMonitoring job %s was not applied because OperatorConfig.collector.kubeletScraping is enabled. kubeletScraping already includes the metrics in this job.", "name", cm.Name)
 			continue
 		}
-		// Reassign so we can safely get a pointer.
```

Comment:
should 444 and 403 also be removed?

Reply:
Actually did you mean to remove this line?

Reply:
Yes, I meant to remove those. 444 and 403 can also be removed, but I'm going to leave them in place for this PR since the linter didn't catch them.

Reply:
Probably because a different name is used rather than `pm := pm`. I removed them in https://github.com/GoogleCloudPlatform/prometheus-engine/pull/1000